catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -330,9 +328,11 @@ The list of alternative file details for the audio (different formats/ quality).
 
 ## Object Shape
 
-JSONb array of dictionaries:
+JSONB array of dictionaries:
 
-`[{"url": "http://example.com/audio.mp3", "filesize": 123456, "bit_rate": 128, "sample_rate": 44100}]`
+```
+[{"url": "http://example.com/audio.mp3", "filesize": 123456, "bit_rate": 128, "sample_rate": 44100}]
```

Comment:
Nit: I'd convert this into a multi-line value just so it renders better in the markdown.
```suggestion
[
    {
        "url": "http://example.com/audio.mp3",
        "filesize": 123456,
        "bit_rate": 128,
        "sample_rate": 44100
    }
]
```